TWLight/templates/base.html

```diff
@@ -215,6 +215,17 @@
         {% endblocktrans %}
       </div>
     {% endif %}
+    {% if not user.userprofile.terms_of_use and not user.is_anonymous %}
```

Comment:
Does this break the "superuser passes all checks" practice?

Reply:
it does. Would you want me to add a superuser override?